overlays/dev/csrf-secure-cookie-patch.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tekton-dashboard
+  namespace: tekton-pipelines
```

Comment:
Would this patch still be applied in the OpenShift dev incase given the namespace is now openshift-pipelines?

Reply:
oh that's a good question... should be able to leave out the namespace here right?